docs/security/pod-sandboxing-on-aks.mdx

```diff
@@ -0,0 +1,318 @@
+    spec:
+      runtimeClassName: kata-mshv-vm-isolation # one line change to make this pod a Kata sandbox.
+```
+
+Let's deploy the updated pod, and run the same tests as before:
+
+- We'll first exec into our pod (`kubectl exec -it <pod-name> -- bash`)
+- Run `capsh --print`. This time around, we should notice much less capabilities show up.
+- Run `mount`. Like `capsh`, we should see considerable less mounted resources than before.
+- Let's attempt to get direct access to host system privileges again, and see what host resources we have access to.
+  - We'll first run `chroot /host-system bash`.
+  - Next, we'll run `crictl pods`. This time around, we should be greeted with an error stating that the file/directory does not exist. From the Kata pod, we won't have access to host resources.
+
+With Kata pods, we can see that we generally have less (or in some cases, no) access to resources that we previously had in a non-Kata pod!
+
+To clean up this first demonstration, run `kubectl delete -f scenarios/system-monitor/` again.
+
+## Exploring pod sandboxing under the hood
+
+As mentioned earlier, Pod Sandboxing runs each pods as a virtual machine on top of your node OS.
+
+Let's take a look under the hood to see how Kata pods are set up.
+
+For this section, let's deploy some sample pods. You can find sample folder for this section under the `docs/getting-started/assets/pod-sandboxing-on-aks` in [AKS labs Github](https://github.com/Azure-Samples/aks-labs/tree/main/docs/security/assets/pod-sandboxing-on-aks/). Please download the folder and place it in the directory you are running the lab on.
+
+```
+kubectl apply -f ./kata_demo/
+```
+
+1. To list your nodes, use the kubectl get nodes command:
+
+```
+kubectl get nodes -o wide
+```
+
+2. Use the kubectl debug command to start a privileged container on your node and connect to it.
+
+```
+kubectl debug node/<node-name> -it --image=mcr.microsoft.com/cbl-mariner/busybox:2.0
+```
+
+In the debug pod, lets now navigate to host kernel and take a look at the Kata VMs:
+
+```
+chroot /host
+ps aux | grep cloud-hypervisor
+ps aux | grep containerd-shim-kata
+```
+
+Very interesting! We can see we have a cloud hypervisor running per Kata container within the host. In Pod Sandboxing, Cloud Hypervisor is the default hypervisor for Kata workloads. This differs from upstream Kata Containers, which uses QEMU as a default.
+
+Lets take a close look at the pods.
+
+```
+sudo crictl pods
+```
+
+We can see all our pods that are on the node. We are mainly interested in the Kata pods. Grab the pod ID for your kata pod, and run the command:
+
+```
+sudo crictl ps -a --pod <kata-pod-id>
+```
+
+This will provide you with the Kata container id. Lets inspect it further. Grab the container-id from the previous output, and run the command:
+
+```
+sudo crictl inspect <container-id> | grep -i runtime
+```
+
+We can see that the runtime is kata based, unlike what you would expect from the normal pod deployments.

```

Comment:
```suggestion
We can see that the runtime is Kata based, unlike what you would expect from the normal pod deployments.
```

Also we should have instruction here to exit the interactive terminal in the node.